fdbserver/storageserver.actor.cpp

```diff
@@ -212,7 +214,11 @@ struct UpdateEagerReadInfo {
 		// SOMEDAY: Theoretically we can avoid a read if there is an earlier overlapping ClearRange
 		if (m.type == MutationRef::ClearRange && !m.param2.startsWith(systemKeys.end))
 			keyBegin.push_back( m.param2 );
-		else if ((m.type == MutationRef::AppendIfFits) || (m.type == MutationRef::ByteMin) || (m.type == MutationRef::ByteMax))
+		else if (m.type == MutationRef::CompareAndClear) {
+			keyBegin.push_back(keyAfter(m.param1, arena));
+			keys.push_back(pair<KeyRef, int>(m.param1, m.param2.size()));
```

Comment:
I think this needs to be m.param2.size() + 1, because if the existing key is a prefix it will compare as equal if we do not get one more byte